jdaviz/configs/default/plugins/data_menu/data_menu.py

```diff
@@ -412,6 +412,7 @@ def set_layer_visibility(self, layer_label, visible=True):
             if layer.layer.label == layer_label:
                 if isinstance(layer, SonifiedDataLayerArtist):
                     layer.audible = visible
+                    layer.visible = visible
```

Comment:
I still don't think this actually does anything and can safely be removed, but please confirm

```suggestion
```